scripts/BuildNugetPackages/Program.cs

```diff
@@ -71,6 +72,16 @@ private static int Main(string[] args)
             return 0;
         }
 
+        private static void PrintSchemaLocations()
```

Comment:
This is a nice little touch 😄

Reply:
Yeah, it'll do until we have a better idea for how to incorporate bundled schema into larger projects :)